examples/chip-tool/main.cpp

```diff
@@ -35,7 +36,9 @@ int main(int argc, char * argv[])
     Commands commands;
 
     registerCommandsEcho(commands);
-    registerClusters(commands);
+    registerPayloadCommands(commands);
```

Comment:
`registerPayloadCommands(commands);` -> `registerCommandsPayload(commands);` to be consistent :)

Reply:
Good point, resolved in the new revision.